backend/lcfs/web/api/compliance_report/repo.py

```diff
@@ -717,38 +667,31 @@ async def get_issued_compliance_units(
 
     @repo_handler
     async def calculate_fuel_quantities(
-        self, compliance_report_id: int, fossil_derived: bool
+        self, compliance_report: ComplianceReport, fossil_derived: bool
```

Comment:
Great! you are passing compliance_report instead of just the id.

Reply:
👍